consensus/parlia/parlia.go

```diff
@@ -1262,6 +1775,30 @@ func encodeSigHeader(w io.Writer, header *types.Header, chainId *big.Int) {
 	}
 }
 
+func encodeSigHeaderWithoutVoteAttestation(w io.Writer, header *types.Header, chainId *big.Int) {
```

Comment:
To me, no need for this func.

Reply:
But I need to distinguish the logic of the SealHash, to make sure not affect other logic. How about making the end position of the extra field a parameter, then we can delete this func?